### PR TITLE
[TRANSLATION]Payment method translation

### PR DESCRIPTION
**Pull request by paullla**

### CODE REVIEW

**Comment by foopang** [comment]
Missing Doctrine migrations script?

**Comment by antonioperic** [comment]
@pjedrzejewski ping

File: src/Sylius/Bundle/PaymentBundle/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -76,6 +76,29 @@ private function addClassesSection(ArrayNodeDefinition $node)
                                         ->scalarNode('choice')->defaultValue('Sylius\Bundle\PaymentBundle\Form\Type\PaymentMethodChoiceType')->end()
```
Comment (pjedrzejewski) [inline src/Sylius/Bundle/PaymentBundle/DependencyInjection/Configuration.php]: Please revert the permissions change.

**Comment by pjedrzejewski** [comment]
Nice work Paula, sorry for the looong waiting. Could you also update data fixtures? (inside FixturesBundle) Thank you very much! :+1: /cc @antonioperic

File: src/Sylius/Bundle/PaymentBundle/Form/Type/PaymentMethodTranslationType.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Sylius\Bundle\PaymentBundle\Form\Type;
```
Comment (stloyd) [inline src/Sylius/Bundle/PaymentBundle/Form/Type/PaymentMethodTranslationType.php]: Missing license header.

File: src/Sylius/Bundle/PaymentBundle/Resources/config/doctrine/model/PaymentMethodTranslation.orm.xml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<doctrine-mapping xmlns="http://doctrine-project.org/schemas/orm/doctrine-mapping">
```
Comment (stloyd) [inline src/Sylius/Bundle/PaymentBundle/Resources/config/doctrine/model/PaymentMethodTranslation.orm.xml]: Missing license header.

File: src/Sylius/Bundle/PaymentBundle/spec/Form/Type/PaymentMethodTranslationTypeSpec.php [hunk 4]
```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: Paula
+ * Date: 24/07/15
+ * Time: 15:59
+ */
```
Comment (stloyd) [inline src/Sylius/Bundle/PaymentBundle/spec/Form/Type/PaymentMethodTranslationTypeSpec.php]: Wrong header & missing license header.

File: src/Sylius/Bundle/TranslationBundle/Doctrine/ORM/TranslatableResourceRepository.php [hunk 5]
```diff
@@ -73,7 +73,6 @@ public function createNew()
         if (!$resource instanceof TranslatableInterface) {
             throw new \InvalidArgumentException('Resource must implement TranslatableInterface.');
         }
-
```
Comment (stloyd) [inline src/Sylius/Bundle/TranslationBundle/Doctrine/ORM/TranslatableResourceRepository.php]: Should be reverted.

File: src/Sylius/Component/Payment/Model/PaymentMethod.php [hunk 6]
```diff
@@ -39,6 +40,8 @@ class PaymentMethod implements PaymentMethodInterface
     protected $description;
 
     /**
+     * Gateway name.
```
Comment (stloyd) [inline src/Sylius/Component/Payment/Model/PaymentMethod.php]: Should be reverted.

File: src/Sylius/Component/Payment/Model/PaymentMethod.php [hunk 7]
```diff
@@ -70,6 +73,7 @@ class PaymentMethod implements PaymentMethodInterface
 
     public function __construct()
     {
+        parent::__construct();
```
Comment (stloyd) [inline src/Sylius/Component/Payment/Model/PaymentMethod.php]: Please add new empty line after this.

File: src/Sylius/Component/Payment/Model/PaymentMethodTranslation.php [hunk 8]
```diff
@@ -0,0 +1,80 @@
+<?php
+namespace Sylius\Component\Payment\Model;
```
Comment (stloyd) [inline src/Sylius/Component/Payment/Model/PaymentMethodTranslation.php]: Missing license header.

File: src/Sylius/Component/Payment/Model/PaymentMethodTranslationInterface.php [hunk 9]
```diff
@@ -0,0 +1,33 @@
+<?php
+namespace Sylius\Component\Payment\Model;
```
Comment (stloyd) [inline src/Sylius/Component/Payment/Model/PaymentMethodTranslationInterface.php]: Missing license header.

File: src/Sylius/Bundle/PaymentBundle/DependencyInjection/Configuration.php (hunk 1 above)
Comment (paullla) [inline src/Sylius/Bundle/PaymentBundle/DependencyInjection/Configuration.php]: @pjedrzejewski Can you please explain what you mean by permissions change?

Comment (pjedrzejewski) [inline src/Sylius/Bundle/PaymentBundle/DependencyInjection/Configuration.php]: File permissions. :)

Comment (paullla) [inline src/Sylius/Bundle/PaymentBundle/DependencyInjection/Configuration.php]: I'm not sure how to revert that :)

**Comment by pjedrzejewski** [comment]
Thank you very much Paula, nice work! :+1:

**Comment by paullla** [comment]
Thank you :)

**Comment by umpirsky** [comment]
@paullla Welcome to Sylius family. Čestitam! :)

**Comment by paullla** [comment]
@umpirsky thank you :)

**Pull request merged**